src/test/java/com/coreos/jetcd/internal/impl/LoadBalancerTest.java

```diff
@@ -0,0 +1,63 @@
+package com.coreos.jetcd.internal.impl;
+
+import com.coreos.jetcd.ClientBuilder;
+import com.coreos.jetcd.KV;
+import com.coreos.jetcd.kv.PutResponse;
+import io.grpc.PickFirstBalancerFactory;
+import io.grpc.util.RoundRobinLoadBalancerFactory;
+import org.testng.Assert;
+import org.testng.annotations.Test;
```

Comment:
Do you have a preference on which test frame work to use? I use Junit, assertJ, and mockito. I hope we can agree on one set of test framework so that it will make code review easier. 

e.g https://github.com/coreos/jetcd/blob/master/src/test/java/com/coreos/jetcd/internal/impl/WatchTest.java